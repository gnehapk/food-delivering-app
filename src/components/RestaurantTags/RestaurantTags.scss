.tag-list {
  display: flex;
  flex-wrap: wrap;

  .tag::after {
    content: "â€¢";
    padding: 0.5rem;
  }
}