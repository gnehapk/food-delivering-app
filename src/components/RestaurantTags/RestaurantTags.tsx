import React from 'react';

import './RestaurantTags.scss';

const RestaurantTags: React.FC<RestaurantTagsPropsType> = (props) => (
  <div className="tag-list">
    {
      props.tags.map((tag, index) => (
        <div className="tag" key={index}> {tag}</div>
      ))
    }
  </div>
);

type RestaurantTagsPropsType = {
  tags: string[];
};

export default RestaurantTags;