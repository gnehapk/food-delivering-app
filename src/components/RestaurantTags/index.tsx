import RestaurantTags from './RestaurantTags';

export default RestaurantTags;