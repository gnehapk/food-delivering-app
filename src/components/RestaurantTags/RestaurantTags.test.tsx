import React from 'react';
import TestRenderer from 'react-test-renderer';
import RestaurantTags from './RestaurantTags';

describe('RestaurantTags', () => {
  it('should render correctly', () => {
    const tree = TestRenderer.create(<RestaurantTags tags={['tag1', 'tag2']} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

