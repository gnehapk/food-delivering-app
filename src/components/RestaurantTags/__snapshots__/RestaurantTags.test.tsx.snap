// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestaurantTags should render correctly 1`] = `
<div
  className="tag-list"
>
  <div
    className="tag"
  >
     
    tag1
  </div>
  <div
    className="tag"
  >
     
    tag2
  </div>
</div>
`;
